modules/currencies/currencies1.template.json

```diff
@@ -1,8 +1,8 @@
 {
     "id": "currencies1",
     "value": "Currencies 1",
-    "main": "<div class=\"container-main\">\r\n    <div class=\"container \">\r\n        <div class=\"row row-header\">\r\n            <div class=\"col-xs-2 col-xs-offset-10 text-center value\">||RATE||<\/div>\r\n        <\/div>\r\n        <div id=\"cycle-container\">\r\n            [itemsTemplate]\r\n        <\/div>\r\n    <\/div>\r\n<\/div>",
-    "item": "<div class=\"row row-finance\">\r\n    <div class=\"col-xs-1 flags\">\r\n        <img class=\"img-circle center-block \" src=\"[CurrencyFlag]\">\r\n    <\/div>\r\n    <div class=\"col-xs-3 value \">[NameShort]<\/div>\r\n    <div class=\"col-xs-2 col-xs-offset-6 text-center value\">[LastTradePriceOnly]<\/div>\r\n<\/div>",
+    "main": "<div class=\"container-main\">\r\n    <div class=\"container \">\r\n        <div class=\"row row-header\">\r\n            <div class=\"col-xs-2 offset-xs-10 text-center value\">||RATE||<\/div>\r\n        <\/div>\r\n        <div id=\"cycle-container\">\r\n            [itemsTemplate]\r\n        <\/div>\r\n    <\/div>\r\n<\/div>",
```

Comment:
Have we also upgraded the Player side bootstrap version then? If so we will likely need to implement a versioning system for that in-case we impact users embedded widgets?